endo_api/settings_prod.py

```
@@ -0,0 +1,65 @@
+from pathlib import Path
+from base_settings import (
+    INSTALLED_APPS,
```

Ruff (0.12.2)

`base_settings.INSTALLED_APPS` imported but unused

Remove unused import

(F401)